packages/fast-components-react-base/src/stack-panel/stack-panel.tsx

```diff
@@ -0,0 +1,738 @@
+import { StackPanelClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { classNames, Direction, RtlScrollConverter } from "@microsoft/fast-web-utilities";
+import { canUseDOM } from "exenv-es6";
+import { get, isEqual, isNil, memoize, merge } from "lodash-es";
+import React from "react";
+import ReactDOM from "react-dom";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import {
+    DisplayNamePrefix,
+    IntersectionObserverEntry,
+    ResizeObserverClassDefinition,
+    ResizeObserverEntry,
+} from "../utilities";
+import {
+    StackPanelHandledProps,
+    stackPanelOrientation,
+    StackPanelProps,
+    StackPanelUnhandledProps,
+} from "./stack-panel.props";
+import { isFunction } from "util";
+
+export interface StackPanelState {
+    disabled: boolean;
```

Comment:
I don't see disabled used anywhere - is it needed?

Reply:
Let's pull it until we figure out we need it.